test/Dialect/FIRRTL/annotations.fir

```diff
@@ -708,316 +657,6 @@ circuit GCTMemTap : %[
 
 ; // -----
 
-; Test sifive.enterprise.grandcentral.ViewAnnotation
-circuit GCTInterface : %[
-[
-  {
-    "class": "sifive.enterprise.grandcentral.GrandCentralView$SerializedViewAnnotation",
-    "name": "view",
```

Comment:
This is now covered in [test/Dialect/FIRRTL/annotations.mlir](https://github.com/llvm/circt/pull/3141/files#diff-c3ad6ada6386fbfc94203d02aec933bfe504a376b9aaa5392d78325eac978b40R188-R226), right? Just to make sure we don't lose the test coverage.